src/Helmet.js

```diff
@@ -270,7 +270,11 @@ const Helmet = (Component) => {
         shouldComponentUpdate(nextProps) {
             return !deepEqual(this.props, nextProps);
         }
-
+        
+        // Component.peak comes from react-side-effect:
+        // For testing, you may use a static peek() method available on the returned component.
+        // It lets you get the current state without resetting the mounted instance stack.
+        // Don’t use it for anything other than testing.
```

Comment:
@cesarandreu I like your idea of shortening it.  You can leave it in the code, just shorten it please.  Thanks!